### PR TITLE
Combine event streams with activations

### PR DESCRIPTION
**Pull request by hsong-rh**

### CODE REVIEW

**Review by benthomasson — CHANGES_REQUESTED** [review]
Don't use BigIntegerField for parent_id.  Use separate foreign key fields for activation and event stream.   In fact you have OneToOneField relations on activation and event stream so you already have the backward reference.

File: tests/integration/api/test_event_stream.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
+#  Copyright 2023 Red Hat, Inc.
```
Comment (mkanoor) [inline tests/integration/api/test_event_stream.py:1]: @hsong-rh 2024?

File: src/aap_eda/api/views/event_stream.py [hunk 2]
```diff
@@ -0,0 +1,323 @@
+#  Copyright 2023 Red Hat, Inc.
```
Comment (mkanoor) [inline src/aap_eda/api/views/event_stream.py:1]: @hsong-rh 2024?

**Comment by Alex-Izquierdo** [comment]
Do we want to continue working in this branch?

**Comment by hsong-rh** [comment]
> Do we want to continue working in this branch?

no, will move all changes to the main branch. Here is a reference.

**Pull request closed without merging**